fail2ban/client/jailsreader.py

```diff
@@ -50,6 +50,7 @@ def jails(self):
 		return self.__jails
 
 	def read(self):
+		self.__jails = list()
```

Comment:
On Mon, 22 Jun 2015, Serg G. Brester wrote:
>    yep, this is interactive only :(

>    see #825, example:
>    fail2ban-client start
>    fail2ban-client -i
>    ... and make reload twice ...

but the bug is within  JailsReader so 
fail2ban/tests/clientreadertestcase.py
would be a good place to test it (theoretically).

P.S. Eventually someone brave will come in to move code base out of
bin/fail2ban-* into the fail2ban module so it could be tested properly
;)

-- 
Yaroslav O. Halchenko, Ph.D.
http://neuro.debian.net http://www.pymvpa.org http://www.fail2ban.org
Research Scientist,            Psychological and Brain Sciences Dept.
Dartmouth College, 419 Moore Hall, Hinman Box 6207, Hanover, NH 03755
Phone: +1 (603) 646-9834                       Fax: +1 (603) 646-1419
WWW:   http://www.linkedin.com/in/yarik